Sources/App/Controllers/TodoController.swift

```diff
@@ -11,16 +11,19 @@ struct TodoController: RouteCollection {
         }
     }
 
+    @Sendable
```

Comment:
Unfortunately this is not what it seems - this just disables sendable checking for the function as opposed to ensuring it's valid. See https://forums.swift.org/t/pitch-inferring-sendable-for-methods/66565/17 and https://forums.swift.org/t/pitch-inferring-sendable-for-methods-and-key-path-literals/68011

Once that's fixed the warning will disappear but until then I don't think we can add it

Reply:
Yeah, it's unfortunate that `@Sendable` doesn't validate the method truly meets the requirements.  It should really have been called `@UncheckedSendable` instead.

In this case, given that these methods are on a struct without any shared mutable state and they only take a `Request`, which itself is `Sendable`, I believe they do in fact meet the requirements to be `Sendable`, so the goal here was just to assert that to the complier and thereby eliminate the warning.

With that said, I like @gwynne's solution better.  Closing in favor of #125.